src/screens/Books/components/ListItem/index.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Text, View, Image } from 'react-native';
+import PropTypes from 'prop-types';
+
+import { CardSection } from '../../../../components/CardSection';
```

Comment:
Use default export in components.

In `CardSection.js` file, do: `export default CardSection;`

And here you do: `import CardSection from ...`

Reply:
Done on the next commit.